bip-0003.md

```diff
@@ -198,17 +197,19 @@ BIP. The idea must be of interest to the broader community or relevant to multip
 and matters concerning only a single project usually do not require standardization and should instead be brought up directly to
 the relevant project.
 
-The authors should first research whether an idea has been considered before. Ideas in Bitcoin are often rediscovered,
+The authors should first research whether their idea has been considered before. Ideas in Bitcoin are often rediscovered,
 and prior related discussions may inform the authors of the issues that may arise in its progression. After some investigation,
-the novelty of an idea can be tested by posting about it to the [Bitcoin Development Mailing
+the novelty and viability of the idea should be tested by posting about the idea to the [Bitcoin Development Mailing
 List](https://groups.google.com/g/bitcoindev). Prior correspondence can be found in the [mailing list
 archive](https://gnusha.org/pi/bitcoindev/).
 
-Vetting an idea publicly before investing the time to describe the idea formally is meant to save both the authors and
-the broader community time. Not only may someone point out relevant discussion topics that were missed in the authors’
+It is recommended that authors establish before or at the start of working on a draft whether their idea may be of
+interest to the Bitcoin community.
+Vetting an idea publicly before investing time and effort to formally describe the idea is meant to save both the authors and
+the community time. Not only may someone point out relevant discussion topics that were missed in the authors’
 research, or that an idea is guaranteed to be rejected based on prior discussions, but describing an idea publicly also
-tests whether it is of interest to more people besides the authors. After establishing that the idea may be of interest
-to the Bitcoin community, the authors should work on drafting a BIP.
+tests whether it is of interest to more people beside the authors.
+Authors should avoid opening a pull request with a BIP draft out of the blue.
```

Comment:
This line is perhaps a bit redundant with the preceding text. (I also don't know how well understood "out of the blue" would be by non-native English readers.)

Reply:
maybe just move that sentence to the start of the previous paragraph? "Vetting an idea publicly..." then serves as an explanation of what "out of the blue" means and gives some explanation of why not to do that?

Reply:
I moved the sentence a bit earlier in the same paragraph. If the Ideation section still feels to redundant, I would be open to shortening it as long as it remains abundantly clear that it is preferred that would-be BIP authors don’t propose a BIP without vetting their idea.